docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1084 @@
+    You can receive the command matching result that the user uttered.
+    And if the registered command is duplicated or the user speaks multiple commands, the recognition result can contain multiple results. In this case, you can select one or reject in the `VoiceControlManagerClient.SetRecognizedCommandsSelectionDelegate()` callbacks
+
+5.  When no longer needed, unset the voice control manager.
+
+    You must disconnect the voice control service and deinitialize the voice control manager using the `Unprepare()` and `Deinitialize()` methods of the [Tizen.Uix.VoiceControlManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.html) class.
+
+The following figure illustrates the voice control manager life-cycle:
+
+**Figure: Voice control manager life-cycle**
+
+![Voice control manager life-cycle](./media/csapi_vc_manager.png)
+
+The user controls the voice control manager service state. In a general scenario:
+1.  The user starts recording for recognition by using a voice control manager application, button, or voice trigger. If the start is successful, the voice control service state changes to `Recording`. For more information on service states, see [Tizen.Uix.VoiceControlManager.ServiceState](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.ServiceState.html) enumration.
+2.  After recording is completed, the service state changes to `Processing` for recognition processing.
+3.  After recognition is completed, the service state changes to `Ready`.
+
+
+
+
+## Prerequisites
+
+To enable your application to use the voice control functionality:
+
+1.  To use the [Tizen.Uix.VoiceControlManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.html) class, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+        <privilege>http://tizen.org/privilege/recorder</privilege>
+        <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+    </privileges>
+    ```
+
+2.  To use the methods and properties of the [Tizen.Uix.VoiceControlManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.html) class, include it in your application:
+
+    ```
+    using Tizen.Uix.VoiceControlManager;
+    ```
+
+    > **Note**   
+    >
+    > To use this privilege, your application must be signed with a platform-level certificate.
+
+3.  Initialize the voice control manager with the `Initialize()` method of the `Tizen.Uix.VoiceControlManager.VoiceControlManagerClient` class:
+
+    ```
+    void InitializeVoiceControlManager()
+    {
+        VoiceControlManagerClient.Initialize();
+    }
+    ```
+
+    If the method call is successful, the voice control state changes to Initialized. The states are defined in the [Tizen.Uix.VoiceControlManager.State](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.State.html) enumeration.
```

Comment:
`Initialized`

Also, referencing https://github.com/Samsung/tizen-docs/pull/971#discussion_r384324217

>Just a question.
>In [Tizen.Uix.VoiceControlManager.State](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.State.html), we have 4 states : Initialized, None, Ready, and Unavailable.
>But in the csapi_vc_manager.png image we only have 3 states : Initialized, None, and Ready.
>In the description we can get to flow from `None` to `Initialized`, from `Initialized` to `Ready`, and vice versa.
>When `Unavailable` state is used?

Reply:
@mijin85cho 
Unavailable state is not the managed state but presenting error occured situation.
By coding rule, getter can not throw the error exception.
So when error is occured on CAPI layer, the getter returns the unavailable state to represent error situation.